first_year_scraper/requirements.txt

```
@@ -0,0 +1,36 @@
+camelot-py==0.8.2
+catkin==0.7.14
+certifi==2020.6.20
+cffi==1.14.2
+chardet==3.0.4
+click==7.1.2
+cryptography==3.1
+et-xmlfile==1.0.1
+gencpp==0.6.0
+geneus==2.2.6
+genlisp==0.4.16
+genmsg==0.5.11
+gennodejs==2.0.1
+genpy==0.6.7
+jdcal==1.4.1
+numpy==1.19.2
+opencv-python==4.4.0.42
+openpyxl==3.0.5
+pandas==1.1.2
+pdfminer.six==20200726
+pluginlib==1.11.3
+pycparser==2.20
+PyPDF2==1.26.0
+python-dateutil==2.8.1
+python-qt-binding==0.3.4
+pytz==2020.1
+qt-gui==0.3.11
+qt-gui-cpp==0.3.11
+rosgraph==1.12.14
+roslib==1.14.4
```

@iakshat I see ROS libs as a dependency here. Which package we have used might have this as a dependency?

---

Surprisingly enough! It was pip. I really don't know why it depends on ROS. I created a fresh new env and installed pip, camelot, opencv and openpyxl. This requirement.txt is generated from that.

---

Strange. As far as I checked pip installs three packages
pip           20.2.3
pkg-resources 0.0.0
setuptools    39.0.1

---

Sorry, there might be problems with my file. I'm exporting python libs for the first time. I'll check again!

---

I see you might be having some issues with pip. Thats not a mistake, ROS is nasty and messes up your packages. I suggest using ROS in dockers. Meanwhile to speed up stuff, I installed the required packages you mentioned and exported the requirements.txt. 

```suggestion
Camelot==12.6.29
chardet==3.0.4
decorator==4.4.2
Elixir==0.7.1
et-xmlfile==1.0.1
jdcal==1.4.1
Jinja2==2.11.2
MarkupSafe==1.1.1
numpy==1.19.2
opencv-python==4.4.0.42
openpyxl==3.0.5
pbr==5.5.0
pkg-resources==0.0.0
six==1.15.0
SQLAlchemy==0.7.10
sqlalchemy-migrate==0.11.0
sqlparse==0.3.1
Tempita==0.5.2
xlrd==0.7.1
xlwt==0.7.2```

---

@iakshat updated list